src/services/dashboards/widgets/cost-trend-stacked/widget-config.ts

```diff
@@ -1,3 +1,35 @@
 import type { WidgetConfig } from '@/services/dashboards/widgets/config';
+import { CHART_TYPE, GROUP_BY } from '@/services/dashboards/widgets/config';
 
-export default {} as WidgetConfig;
+const costTrendStackedWidgetConfig: WidgetConfig = {
+    widget_config_id: 'costTrendStacked',
+    base_configs: [{ config_id: 'baseTrend' }],
+    title: 'Cost Trend Stacked',
+    labels: ['Cost'],
+    description: {
+        translation_id: 'DASHBOARDS.WIDGET.COST_TREND_STACKED.DESC',
+        preview_image: 'xxx.png',
+    },
+    scopes: ['PROJECT', 'WORKSPACE'],
+    theme: {
+        inherit: true,
+    },
+    sizes: ['lg', 'full'],
+    widget_options: {
+        granularity: 'MONTHLY',
+        enable_legend: false,
+        chart_type: CHART_TYPE.STACKED_COLUMN,
+    },
```

Comment:
I think it's bit different from the widget spec in the widget sheet.
<img width="1589" alt="image" src="https://user-images.githubusercontent.com/26986739/203884685-bf48f12d-090f-4c3a-8082-f108b1ef4faa.png">
If this widget needs option for granularity, please update the widget sheet to sync and manage it.

Reply:
1. options X / schema X
2. **options O / schema X**
3. options X / schema O (impossible)
4. options O / schema O

This case is 2. So I'll add `granularity` to options in widget sheet.